examples/bcoin-getTrustedInput.js

```diff
@@ -1,8 +1,9 @@
 'use strict';
 
+const TX = require('bcoin/lib/primitives/tx');
```

Comment:
Do Object Restructuring imports not work with bcoin now? (e.g. `const {TX} = require('bcoin');`)

Reply:
I don't want to import more stuff than necessary, not sure how `webpack` resolves this kind of dependencies.
Also these primitives may get separate in the future.